bitnami/airflow/values.yaml

```diff
@@ -365,9 +374,38 @@ metrics:
   # tolerations: []
 
   ## Metrics exporter pod Annotation and Labels
+  # podAnnotations: {}
   # podLabels: {}
 
   podAnnotations:
     prometheus.io/scrape: "true"
     prometheus.io/port: "9112"
 
+additionalRepos:
+  enabled: false
+  # repos:
+  # - name:
+    # repository:
+    # branch:
+    # interval:
+    # sshKeySecret:
+workers:
+  # - name:
+    #volumes:
+    #- name:
+    #  emptyDir: {}
+    #image:
+    #  registry:
+    #  repository:
+    #  tag:
+    #  pullPolicy:
+    # additionalRepos:
+      # enabled: false
+      # repos:
+      # - name:
+      #   repository:
+      #   branch:
+      #   interval:
+      #   sshKeySecret:
```

Comment:
What about the following approach:
```yaml
      ## SSH keys to clone the repo
      ## 
      # sshKeys:
      ## Name of existing secret containing the SSH keys to clone the repo
      ## When provided, `workers[0].additionalRepos.repos[0].sshKeys` will be ignored
      ##
      # sshKeysSecretName:
```
Then you can create a **ssh-keys-secrets.yaml** manifest like the one below:
```yaml
{{- range .Values.workers }}
{{- if .additionalRepos.enabled }}
{{- range .additionalRepos.repos }}
{{- if empty .sshKeysSecretName }}
apiVersion: v1
kind: Secret
metadata:
  name: {{ .name }}
  labels:
    app.kubernetes.io/name: {{ include "airflow.name" $ }}
    helm.sh/chart: {{ include "airflow.chart" $ }}
    app.kubernetes.io/instance: {{ $.Release.Name }}
    app.kubernetes.io/managed-by: {{ $.Release.Service }}
data: 
  ssh-keys: {{ .sshKeys | b64enc }}
---
{{- end }}
{{- end }}
{{- end }}
{{- end }}
```

Reply:
I'm not sure I understand here. Are you saying we could include the sshkeys directly here, and then have it generate the secrets with those keys, as opposed to how I have it set up now where it's assumed that the ssh keys are already a secret, and you tell it which secret they're in?

Reply:
Exactly. You support both taken the ssh keys from an already existing secret, and passing them in the values so you create the secret

Reply:
I think this assumes that you're definitely supplying ssh keys to every repo eh? I made an enabled option to turn it on in case you don't need ssh keys.